modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -62,6 +76,29 @@ do
 done
 ----
 
+. After the nodes in your cluster have rebooted, start all of the machine configuration pools:
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool master --type='merge' --patch \
+  '{ "spec": { "paused": false } }'
+----
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool worker --type='merge' --patch \
+  '{ "spec": { "paused": false } }'
+----
+
+. To confirm the status of the new machine configuration on the hosts, enter the following command.
++
+[source,terminal]
+----
+$ oc describe <node> | egrep "hostname|machineconfig"
+----
++
+When the new configuration is applied, the value of `machineconfiguration.openshift.io/state` field is `Done`.
```

Comment:
Users shall check whether the `machineconfiguration.openshift.io/currentConfig` of all the nodes from one machine config pool is the same and is the name of the latest MachineConfig.